adhocracy-plus/config/settings/base.py

```diff
@@ -555,3 +555,8 @@
 
 # Add insights for project if insight model exists
 INSIGHT_MODEL = "a4_candy_projects.ProjectInsight"
+
+# Celery configuration
+CELERY_BROKER_URL = "redis://localhost:6379"
```

Comment:
If we add them here testing locally will produce an error every time a celery task is used. We have to add `CELERY_TASK_ALWAYS_EAGER = True` to `dev.py` so it doesn't require a broker/backend.

Reply:
done